mne/viz/evoked.py

```diff
@@ -2116,12 +2141,20 @@ def plot_compare_evokeds(evokeds, picks=None, gfp=None, colors=None,
     # build up evokeds into a dict, if it's not already
     if isinstance(evokeds, Evoked):
         evokeds = [evokeds]
+    elif isinstance(evokeds, Epochs):
+        evokeds = {1: list(evokeds.iter_evoked())}
+
     if isinstance(evokeds, (list, tuple)):
         evokeds = {str(idx + 1): evk for idx, evk in enumerate(evokeds)}
     if not isinstance(evokeds, dict):
         raise TypeError('"evokeds" must be a dict, list, or instance of '
                         'mne.Evoked; got {}'.format(type(evokeds).__name__))
+
     evokeds = deepcopy(evokeds)  # avoid modifying dict outside function scope
+
+    # this line will return unmodified if no epochs in there
+    evokeds = _convert_epochs_to_evoked_for_pce(evokeds, Epochs)
```

Comment:
I don't understand this helper function. What is the `_for_pce` suffix? Why do we need to pass a class to the function? are we modifying it?

maybe 
```py
if isinstance(evokeds, Epochs):
   evokeds = _make_evoked_from_epochs(evokeds)
```

But this call shows that maybe we should change the parameter name.